pkg/sentry/socket/plugin/cgo/BUILD

```diff
@@ -0,0 +1,29 @@
+package(licenses = ["notice"])
+
+load("//tools:defs.bzl", "go_library")
+
+go_library(
+    name = "cgo",
+    srcs = [
+        "socket_unsafe.go",
+        "stack_unsafe.go",
+        "util_unsafe.go",
+    ],
+    cgo = True,
+    visibility = ["//visibility:public"],
+    deps = [
+        "//pkg/abi/linux",
+        "//pkg/abi/linux/errno",
+        "//runsc/config",
+    ],
+    cdeps = [
+        "@libpluginstack//:libpluginstack",
+    ],
+    copts = [
+        "-march=native",
+        "-I external/libpluginstack/lib/libtle_glue",
+    ],
+    clinkopts = [
+        "-L external/libpluginstack",
+    ],
```

Comment:
After #10938 is submitted, please rename the arguments as follows:

* `cdeps` -> `cgo_cdeps`
* `copts` -> `cgo_copts`
* `clinkopts` -> `cgo_clinkopts`

Then also add a new argument called `bazel_cgo`, set depending on the `config_setting` like so:

```
    bazel_cgo = select({
        ":network_plugins": True,
        "//conditions:default": False,
    }),
```